docs/admin-ports.md

```diff
@@ -37,6 +37,7 @@ The following is a list of ports used by internal DC/OS services, and their corr
  - 15101: dcos-marathon libprocess
  - 15201: dcos-metronome libprocess
  - 61053: dcos-mesos-dns
+ - 61825: dcos-telegraf
```

Comment:
We're listening on UDP 61825, not TCP, so this is in the wrong section. This should be added to the end of the list beneath the following `UDP` section.

Reply:
oops

Reply:
Oh, and this should be added to the `UDP` section under `All Roles`, since we're reserving the port on all nodes.